src/clr-icons/clr-icons-api.ts

```diff
@@ -63,6 +64,16 @@ export class ClarityIconsApi {
     }
   }
 
+  init(options: any = { includeCore: true }) {
```

Comment:
I think it's better to use an interface here instead of this `any` type.

Reply:
Yeah. We made a change to actually pass in any set optionally for init(). You can take a look at the latest changes. The reason for this change is.. if we explicitly have a shapeSet like the way I had it initially it doesn't tree-shake the way we want it to.